packages/core/package.json

```diff
@@ -11,28 +11,42 @@
   "collection:main": "dist/collection/index.js",
   "unpkg": "dist/dso-toolkit/dso-toolkit.js",
   "files": [
-    "dist/",
-    "loader/"
+    "/dist/!(**/*.stories.*)",
```

Comment:
Dit zorgt ervoor dat de gecompileerde stories niet in de gepubliceerde package naar NPM meekomen.

In `tsconfig.json`'s `exclude` plaatsen werkt niet, want dan negeert je editor deze file ook..

Note: Misschien `tsconfig.local.json` gebruiken om weer te includen? Zodat Stencil ook de stories niet compileert